collection/rancher/v2.x/profile-collector/continuous_profiling.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-# Which app to profile? Supported choices: rancher, cattle-cluster-agent
+set x
```

Comment:
this line seemed to break the ability to use the options and made the script automatically run with only collecting rancher profiles